protocols/identify/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 - Update to `libp2p-swarm` `v0.36.0`.
 
+- Expose explicits errors via `UpgradeError` instead of generic `io::Error`.
+
```

Comment:
```suggestion
- Expose explicits errors via `UpgradeError` instead of generic `io::Error`. See [PR 2630].

[PR 2630]: https://github.com/libp2p/rust-libp2p/pull/2630
```